org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/handlers/BuildWorkspaceHandler.java

```diff
@@ -119,7 +123,14 @@ else if (projectsManager.isBuildFile(file)) {
 				connection.publishDiagnostics(new PublishDiagnosticsParams(ResourceUtils.toClientUri(uri), diagnostics));
 			}
 		}
+	}
 
-
+	private static String convertMarker(IMarker marker) {
+		StringBuilder builder = new StringBuilder();
+		String message = marker.getAttribute(IMarker.MESSAGE, "");
+		String code = String.valueOf(marker.getAttribute(IJavaModelMarker.ID, 0));
+		builder.append("message:" + message);
```

Comment:
you need to add some space, new line or a separator between the message attributes